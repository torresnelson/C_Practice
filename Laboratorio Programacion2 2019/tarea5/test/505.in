# prueba de heap: reducir, hay_valor
hay_valor 0
hay_valor 5
insertar_en_heap (5,trr)
hay_valor 5
menor
insertar_en_heap (8,kop)
insertar_en_heap (6,hhy)
hay_valor 5
hay_valor 8
hay_valor 4
reducir 8
hay_valor 8
hay_valor 4
menor
eliminar_menor
hay_valor 4
menor
insertar_en_heap (14,xsw)
insertar_en_heap (2,lop)
insertar_en_heap (7,dfrt)
insertar_en_heap (16,bhu)
insertar_en_heap (17,esw)
insertar_en_heap (20,jiu)
insertar_en_heap (8,t55)
es_lleno_heap
insertar_en_heap (18,i98u)
es_lleno_heap
reducir 8
hay_valor 4
menor
eliminar_menor
menor
reducir 6
menor
Fin
