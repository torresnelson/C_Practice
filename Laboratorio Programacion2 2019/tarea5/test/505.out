1>#  prueba de heap: reducir, hay_valor.
2>No hay valor.
3>No hay valor.
4>Se insertó el elemento.
5>Hay valor.
6>(5,trr)
7>Se insertó el elemento.
8>Se insertó el elemento.
9>Hay valor.
10>Hay valor.
11>No hay valor.
12>Se redujo el valor.
13>No hay valor.
14>Hay valor.
15>(4,kop)
16>Eliminado.
17>No hay valor.
18>(5,trr)
19>Se insertó el elemento.
20>Se insertó el elemento.
21>Se insertó el elemento.
22>Se insertó el elemento.
23>Se insertó el elemento.
24>Se insertó el elemento.
25>Se insertó el elemento.
26>No lleno.
27>Se insertó el elemento.
28>LLeno.
29>Se redujo el valor.
30>Hay valor.
31>(2,lop)
32>Eliminado.
33>(4,t55)
34>Se redujo el valor.
35>(3,hhy)
36>Fin.
